src/RightsService.Models.Db/DbUser.cs

```diff
@@ -11,6 +11,7 @@ public class DbUser
 
         public Guid Id { get; set; }
         public ICollection<DbUserRight> Rights { get; set; }
```

Comment:
I think now user does not have rights, only roles, ask Spartak about it

Reply:
done